### PR TITLE
fix: change order by for organization_has_mobile_app_events

### PR DESCRIPTION
**Pull request by scefali**
1. Order by timestamp to make the query more efficient
2. Remove log statement

### CODE REVIEW

File: src/sentry/api/endpoints/organization_has_mobile_app_events.py [hunk 1]
```diff
@@ -43,8 +43,8 @@ def _get(self, request, organization):
             # TODO: update discover query for null browser and client_os_name is android or ios
             result = discover.query(
                 query=query,
-                orderby="browser.name",
-                selected_columns=["browser.name", "project", "id", "client_os.name"],
+                orderby="-timestamp",
+                selected_columns=["browser.name", "client_os.name", "timestamp"],
```
Comment (lynnagara) [inline src/sentry/api/endpoints/organization_has_mobile_app_events.py:47]: Do you need to select the timestamp? It doesn't seem to be used.

Comment (scefali) [reply src/sentry/api/endpoints/organization_has_mobile_app_events.py:47]: @lynnagara I got an error saying I can't order by a field that isn't being selected

File: src/sentry/api/endpoints/organization_has_mobile_app_events.py [hunk 2]
```diff
@@ -43,8 +43,8 @@ def _get(self, request, organization):
...
                 limit=1,
                 params={
                     "start": timezone.now() - timedelta(days=1),
```
Comment (lynnagara) [inline src/sentry/api/endpoints/organization_has_mobile_app_events.py:50]: Just out of curiosity is this only supposed to tell you if one event with a non null browser name field was sent in the last day? We are intentionally excluding results earlier than this window?

Comment (scefali) [reply src/sentry/api/endpoints/organization_has_mobile_app_events.py:50]: @lynnagara It's supposed to tell us if we have an event that matches a set of 4 browsers. I just picked the last 24 hours cause we do that for the global SDK updates; it's somewhat arbitrary.

File: src/sentry/api/endpoints/organization_has_mobile_app_events.py (hunk 1 above)
Comment (lynnagara) [reply src/sentry/api/endpoints/organization_has_mobile_app_events.py:47]: Ok, must be a restriction of the discover module

File: src/sentry/api/endpoints/organization_has_mobile_app_events.py (hunk 2 above)
Comment (lynnagara) [reply src/sentry/api/endpoints/organization_has_mobile_app_events.py:50]: Fair enough

**Review by lynnagara — APPROVED** [review]

**Pull request merged**